src/aiai_eval/sequence_classification.py

```diff
@@ -137,14 +137,38 @@ def _load_data_collator(self, tokenizer: PreTrainedTokenizerBase):
         """
         return DataCollatorWithPadding(tokenizer, padding="longest")
 
-    def _get_spacy_predictions_and_labels(self, model, dataset: Dataset) -> tuple:
+    def _extract_spacy_predictions(self, tokens_processed: tuple) -> list:
+        """Helper function that extracts the predictions from a SpaCy model.
+        Aside from extracting the predictions from the model, it also aligns the
+        predictions with the gold tokens, in case the SpaCy tokeniser tokenises the
+        text different from those.
```

Comment:
```suggestion
        """Helper function that extracts the predictions from a SpaCy model.
        
        Aside from extracting the predictions from the model, it also aligns the
        predictions with the gold tokens, in case the SpaCy tokeniser tokenises the
        text different from those.
```